### PR TITLE
IDEX-3126: Provide path to the projects sources directory with metada…

### PR DESCRIPTION
**Pull request by garagatyi**
…ta of a dev machine

### CODE REVIEW

**Comment by skabashnyuk** [comment]
LGTM

File: platform-api/che-core-api-machine/src/main/java/org/eclipse/che/api/machine/server/MachineService.java [hunk 1]
```diff
@@ -511,7 +512,8 @@ private MachineDescriptor toDescriptor(Instance machine) throws ServerException
                                             .withRef(serverEntry.getValue().getRef())
                                             .withUrl(serverEntry.getValue().getUrl()));
         }
-        machineDescriptor.withMetadata(machine.getMetadata().getProperties())
+        machineDescriptor.withProperties(machine.getMetadata().getProperties())
+                         .withMetadata(newDto(MachineMetadataDto.class).withEnvVariables(machine.getMetadata().getEnvVariables()))
```
Comment (evoevodin) [inline platform-api/che-core-api-machine/src/main/java/org/eclipse/che/api/machine/server/MachineService.java]: I think that <code>.withEnvVariables(machine.getMetadata().getEnvVariables()))</code> should be in a new line

Comment (skabashnyuk) [inline platform-api/che-core-api-machine/src/main/java/org/eclipse/che/api/machine/server/MachineService.java]: +1

**Comment by evoevodin** [comment]
:+1: Good job!

**Pull request merged**